tests/e2e_tests/cypress/integration/04-deviceDetails.spec.js

```diff
@@ -29,11 +29,9 @@ context('Layout assertions', () => {
         () =>
           cy
             .get('.deviceListItem')
-            .click()
             // the deviceconnect connection might not be established right away,
             // due to polling intervals the fastest way to refresh is collapsing & expanding again...
```

Comment:
does this comment still make sense?

Reply:
Yep, the clicking will still be done repeatedly, but now just once and then wait until the animation had settled, Check if the button is usable and repeat